libs/atqr/domain/src/lib/domain-spec/services/Deadline Monitor.disl

```diff
@@ -13,106 +13,92 @@ domain ATQR
        */
       ⏰ NewDayStarted!
         for each :challenge in Challenges
-          when .deadline is before Today
-            publish DeadlineReached! :challengeId
+          when .deadline is after Today
+          set it's @status to .Overdue
+            publish TimeToOfficiateArrived! :challengeId
 
           // TODO: Check which logic is better for sending emails
           // This or the one inside NotificationService (aren't they both the same?)
 
           //TODO: add logic for JudgeForgotToAcceptRequest and JudgesLastChancetoAccepRequest AND 2 more logics: JudgeForgotToOfficiate and JudgesLastChanceToOfficiate
-        ->ContenderForgotToPlay! :challenge
-            when .enrollmentDeadline is Today
-              and .createdAt is before 2 days ago
-              list all .invitees that
-                - have not enrolled in :challengeId
-                - have not been notified
+
+          when .enrollmentDeadline is after Today
+            and .createdAt is before 2 days ago
+            list all .invitees that
+              - have not enrolled in :challengeId
+              - have not been notified
+
+          if list is not empty
+            publish ContenderForgotToPlay! :challengeId :list
+
+          when .officiationDeadline is after Today
+            and .createdAt is before 2 days ago
+            list all .invitees that
+              - have not entered Officiation in :challengeId
+              - have not been notified
 
             if list is not empty
-              publish ReminderToPlay! :challengeId :list
+              publish JudgeForgotToPlay! :challengeId :list
+
+          when .createdAt is before 5 days ago
+            and .enrollmentDeadline is after Today
+            list all .invitees that
+              - have not enrolled in :challengeId
+              - have been notified at most once
+
+          if list is not empty
+            publish LastChanceToPlayArrived! :challengeId :list
```

Comment:
This event doesn't explain enough to me what happened. Try to follow the pattern that was established in the other events.

Reply:
This one is referred to both players: contenders and judges, and that's why it doesn't specify which one, since it will be send to both.